src/components/table/tableTypes.ts

```diff
@@ -17,6 +19,12 @@ export interface ParaHeadCell extends HeadCell {
   id: keyof Para;
 }
 
+export interface IepHeadCell extends HeadCell {
+  id: keyof Iep;
+}
+
+export type StudentWithIepHeadcell = StudentHeadCell | IepHeadCell;
+
 export function isStudent(person: Student | Para): person is Student {
   return (person as Student).student_id !== undefined;
 }
```

Comment:
Since we're using this function to test for StudentWithIep, we should convert it to test for that specifically. 
```suggestion
export function isStudentWithIep(person: StudentWithIep | Para): person is StudentWithIep {
  return (person as StudentWithIep).student_id !== undefined && (person as StudentWithIep).iep_id !== undefined;
}
```